### ISSUE TITLE
DHCP Implementation is too minimal 

### ISSUE THREAD

**Issue by PherricOxide** [DataSoft/Honeyd#12]
Honeyd's DHCP currently ignores the lease time (and a bunch of other DHCP params...), which means after the lease expires it'll keep hanging on to the IP even though it'll likely be assigned to someone else as well.
# Things to do

Parse lease time and any other useful stuff out of DHCP ack packets 
Attempt to renew address if the lease time is coming up soon (1/2 lease time is standard for a renew attempt)
Try and release/lease if the renew fails

Basically, implement all the lease parts of the protocol that are currently being ignored.

**Comment by PherricOxide** [comment]
This was mostly fixed, but never closed. Honeyd attempts to renew the addresses now.
